influxdb3_server/src/query_executor.rs

```diff
@@ -840,3 +854,117 @@ fn from_query_log_entries(
     let batch = RecordBatch::try_new(schema, columns)?;
     Ok(batch)
 }
+
+struct LastCachesTable {
```

Comment:
Might be good to pull this out of the query_executor and into its own file. Create a structure for different system tables to be defined. I imagine we'll have a number of these over time.

Reply:
I moved out the system table related code to its own module in https://github.com/influxdata/influxdb/pull/25166/commits/7c1f4db1eea45f8b12cc6a013260fc668c658fc9 - 🧹 definitely cleaner.

No code was changed and CI is ✅ so I will get this merged.